### PR TITLE
TOMEE-2326 - Define description and rules to create new examples

### PR DESCRIPTION
**Pull request by brunobat**
Signed-off-by: brunobat <brunobat@gmail.com>

### CODE REVIEW

File: examples/README.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+= TomEE Examples
+
+These examples demonstrate particular features of the JEE ecosystem to help
```
Comment (jeanouii) [inline examples/README.adoc]: JEE should be Java EE
JEE is not an authorized acronyme.

File: examples/README.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+developers in the creation of their own applications.
+
+The idea is that each example focuses in a very specific framework feature and
+demonstrates it's usage and behaviour in a simple way.
```
Comment (jeanouii) [inline examples/README.adoc]: it's --> its

File: examples/README.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+== Rules to create a new example.
+
+* Demo just one feaure with simple business logic.
```
Comment (jeanouii) [inline examples/README.adoc]: feaure --> feature

File: examples/README.adoc (hunk 1 above)
Comment (brunobat) [reply examples/README.adoc]: Thanks, Done

File: examples/README.adoc (hunk 2 above)
Comment (brunobat) [reply examples/README.adoc]: thanks. Done

File: examples/README.adoc (hunk 3 above)
Comment (brunobat) [reply examples/README.adoc]: thanks. Done

**Comment by jeanouii** [comment]
Ok looks good. I'll merge it when my current build is done.

**Pull request merged**